### ISSUE TITLE
Not working on optimism => not stats for optimism

### ISSUE THREAD

**Issue by martinkrung** [Uniswap/uniswap-v3-subgraph#47]
![Screenshot at 2021-08-24 22-42-42](https://user-images.githubusercontent.com/918180/130687134-a83e7e7b-83aa-4367-9c2d-75cfbd793ef8.png)

https://thegraph.com/legacy-explorer/subgraph/ianlapham/uniswap-optimism

![Screenshot at 2021-08-22 20-05-04](https://user-images.githubusercontent.com/918180/130687041-67871f4a-aaa9-404e-9c19-7f2b445acdae.png)

**Comment by Devnant** [comment]
Yeah I'm blind about pool stats on Optimism. Don't have a clue about volume and I'm afraid this is hurting the DEX. Been going blind for a few days now. Please fix this.

**Comment by ianlapham** [comment]
its an issue with the graph node itself working through it with the team - quick summary is that the queries to the http endpoint are referencing a different subgraph than whats displayed on the sandbox - data is pretty unusable
